### DISCUSSION TITLE
not find coins for the transaction

### DISCUSSION THREAD

**Discussion by Atikin1416** [Chia-Network/chia-blockchain#17514] [Support]
Hello. My Mac version 10.15.7 (19H2026) stopped supporting the chia wallet installed 2022 (Chia-1.3.0.dmg.) Downloaded the Chia wallet-2.1.3 my mac does not run it. I had to download a wallet for windows 10, going into my wallet I did not find coins for the transaction 0xd48f84c3ef8b05634aca893efb07e177c5704e4e232d1525992a9bcfbef7a826. What could this be related to?

**Comment by BrandtH22** [comment]
Hey @Atikin1416 , if you are seeing all of your other transactions then I would resync the wallet with the GUI settings / advanced / resync option. This will close the gui and upon reopening will resync the wallet. 

If you are using the cli than you may want to stop chia, rename the wallet db at `~/.chia/mainnet/wallet/db/`. Upon restarting chia it will sync from scratch. 

If the other transaction is still not present then I would verify the mnemonic for it is correct by confirming the associated xch address with either the guis settings / advanced / create signature option or with the cli `chia keys derive search -t address xch1fc9hat6jf5w0eplw0ps2k7qftmtqt3wc34ehrjmayhznpja4e9qqzf5p4t` .

More instructions for the GUI version of this confirmation, note these directions use an NFT id but the same can be done with an xch address. Essentially if it creates a valid signature / that xch address is found in the derive search command then it is a syncing issue. If it does not create a valid signature / is not found in the derive search command then we need to determine which mnemonic that xch address is associated:
https://docs.chia.net/faq/#how-can-i-sign-and-verify-a-message-with-an-nft

Let us know if any questions arise and take note that we can generally provide more timely and thorough support in our discord server https://discord.gg/chia

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
